kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -328,11 +328,13 @@ define([
         }
 
         function syncDataModel() {
-            // map structure of data into an array of parameter objects
-            const dataValues = dataModel.rowOrder.map((rowId) => dataModel.rows[rowId].values);
-            paramsBus.emit('sync-data-model', {
-                values: dataValues,
-            });
+            if (!viewOnly) {
```

Comment:
This triggers a number of updates to the overall cell state (and a parameter evaluation) that are not necessary when in view only mode.